content/press.adoc

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: "Jenkins Press Information"
+---
+
+
+If you're a journalist, blogger or anybody else who might consider themselves
+"press" this page should help you refer to, explain or otherwise cite the
+Jenkins project correctly. If this page does not sufficiently answer a question
+you may have, or you're seeking comment, please reach out to one of our <<Press Contacts>> below.
+
+
+== About Jenkins
+
+
+.Elevator Pitch
+""
+Jenkins is an open source automation server which enables developers around the
+world reliably build, test and deploy their software
+""
+
+.Blurb
+""
+Jenkins, originally founded in 2006 as "Hudson", is one of the leading
+automation servers available. Using an extensible, plugin-based architecture
+developers have created hundreds of plugins to adapt Jenkins to a multitude of
+build, test, and deployment automation workloads. In 2015, Jenkins surpassed
+100,000 known installations making it the most widely deployed automation server.
+""
+
+
+=== Governance
+
+Our
+link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Document[Governance
+Document] outlines the project's philosophy, structure and guidelines. This
+also includes information on the
+link:https://wiki.jenkins-ci.org/display/JENKINS/Governance+Board[Governance
+Board], which is the group with acting executive authority. While board members
+are available for comment, we encourage you to reach out to our <<Press Contacts>>
+as an initial point of contact for press inquiries.
+
+
+The Jenkins project is part of link:http://spi-inc.org/[Software in the Public
+Interest], a 501 (c) (3) non-profit organization, which holds the Jenkins
+trademark and acts as a legal umbrella organization.
+
+=== Branding
+
+We ask that the following branding guidelines be used when referring to or
+writing about the Jenkins project:
+
+*Correct:*
+
+* the Jenkins project
+* Jenkins
+
+*Incorrect:*
+
+* Jenkins CI
+* Jenkins server
+* Mr. Jenkins
+
+
+==== Artwork
+
+image::/images/logo.png[title="Jenkins", width="128", float=right]
+
+On our link:https://wiki.jenkins-ci.org/display/JENKINS/Logo[Logo] page we have
+documented some versions of our logo, including the color palette our branding
+uses. Our logos can be downloaded in a number of sizes, or as vector graphics,
+from link:http://mirrors.jenkins-ci.org/art/[here].
+
+When embedding our logo in an article, we prefer the `logo.png` (see Figure 1)
+to be used over the `headshot.png` image.
+
+If you have any questions about artwork, please ask one of the <<Press Contacts>>.
+
+== Press Contacts
+
+The following individuals can be reached out to for comment, clarification
+or questions about the Jenkins project. Please bear in mind that many are
+volunteers so availability and time may be limited.
+
+=== US press contacts
```

Comment:
Doesn't hurt, I think.  Different publications in different countries; timezone coverage in case people want (to attempt getting) a particularly speedy response, etc.